pkg/disttask/framework/handle/handle.go

```diff
@@ -286,6 +286,35 @@ func GetCloudStorageURI(ctx context.Context, store kv.Storage) string {
 	return cloudURI
 }
 
+// GetTaskMgrToAccessDXFService returns the task manager to access DXF service.
+func GetTaskMgrToAccessDXFService() (*storage.TaskManager, error) {
+	// TODO currently DXF service is not fully implemented, so we always return
+	// task manager of current keyspace, replace it with below code when DXF service is ready.
+	return storage.GetTaskManager()
+}
+
+//// GetTaskMgrToAccessDXFService returns the task manager to access DXF service.
```

Comment:
will uncomment this part when we are ready to switch to DXF service

Reply:
If there is a plan to abandon the DXF service in user keyspace, I think we don't need to load the system DXF service every time? Simply initialize the only `GetTaskManager` to the DXF service one is enough.

Reply:
yes, we can do it later to avoid block the e2e test, right now, import-into only runs across keyspace